.github/actions/elasticsearch/README.md

```diff
@@ -0,0 +1,19 @@
+# Elasticsearch GitHub Action
```

Comment:
currently this action is embedded but we can pull it out into it's own repo to use elsewhere